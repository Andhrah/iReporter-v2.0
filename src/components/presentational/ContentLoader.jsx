import React from 'react';
import ContentLoader from 'react-content-loader';

const Loader = () => (
  <div className="grid-container" style={{ marginBottom: 30 }}>
    <ContentLoader
      height={230}
      width={300}
      speed={3}
      primaryColor="#f3f3f3"
      secondaryColor="rgba(217, 196, 224, 0.32)"
    />
  </div>
);

export default Loader;
