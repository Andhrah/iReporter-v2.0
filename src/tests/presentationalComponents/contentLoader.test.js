import React from 'react';
import { shallow } from 'enzyme';
import ContentLoader from '../../components/presentational/ContentLoader';

describe('ContentLoader  test suite', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ContentLoader />);
    expect(wrapper.length).toBe(1);
    expect(wrapper).toMatchSnapshot();
  });
});
