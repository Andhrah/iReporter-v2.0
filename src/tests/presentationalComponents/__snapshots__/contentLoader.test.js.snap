// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentLoader  test suite renders correctly 1`] = `
<div
  className="grid-container"
  style={
    Object {
      "marginBottom": 30,
    }
  }
>
  <ContentLoader
    height={230}
    primaryColor="#f3f3f3"
    secondaryColor="rgba(217, 196, 224, 0.32)"
    speed={3}
    width={300}
  />
</div>
`;
